VRDR.Messaging/VRDRMessaging.xml

```diff
@@ -29,10 +29,32 @@
             <value>the message id.</value>
         </member>
         <member name="P:VRDR.AckMessage.BlockCount">
+            DELETE THIS   vvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
```

Comment:
I'm rather confident that this file is autogenerated, so there should be no need to do anything special here if the actual code is updated.